app/gateway/changelog.md

```diff
@@ -5,6 +5,24 @@ no_version: true
 
 <!-- vale off -->
 
+## 3.2.2.0
+**Release Date** 2023-03-15
+
+### Fixes 
+#### Enterprise
+* In Kong 3.2.1.0 and 3.2.1.1, `alpine` and `ubuntu` ARM64 artifacts incorrecty handled HTTP2 requests, causing the protocol to fail. These artifacts have been removed. 
+* In all 3.x versions of Kong Gateway prior to this release, the default logrotate file `/etc/logrotate.d/kong-enterprise-edition` was missing. 
+
+#### Plugins
+* SAML: 
+    * The SAML plugin now works on read-only file systems.
+    * The SAML plugin can now handle the field `session_auth_ttl` (removed since 3.2.0.0).
+
+* Datadog Tracing plugin: We found some late-breaking issues with the Datadog Tracing plugin and elected to remove it from the 3.2 release. We plan to add the plugin back with the issues fixed in a later release. 
+
+### Known issues
+* Due to changes in GPG keys, customers who use yum to install this release will get a `Public key for kong-enterprise-edition-3.2.1.0.rhel7.amd64.rpm is not installed` error. The package is signed, it is just signed with a different (rotated) key from the metadata service, which triggers the error in yum. In order to avoid this error, manually download the package from [download.konghq.com](https://download.konghq.com/) and then install it. 
```

Comment:
```suggestion
* Due to changes in GPG keys, using yum to install this release triggers a `Public key for kong-enterprise-edition-3.2.1.0.rhel7.amd64.rpm is not installed` error. The package *is* signed, however, it's signed with a different (rotated) key from the metadata service, which triggers the error in yum. To avoid this error, manually download the package from [download.konghq.com](https://download.konghq.com/) and install it. 
```
We don't generally say "customers" in the changelog. "Users" at most, though we don't even need to say that here either.

Also avoid using "in order to", please just say "to".